src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_executable.py

```diff
@@ -19,16 +17,9 @@
 @pytest.mark.cli_test
 @pytest.mark.e2etest
 class TestExecutable:
-    @pytest.mark.skipif(
```

Comment:
Did we fix the exit code 4294967295 in this PR?

Reply:
The test will run pyinstaller package process in the latest code, but will not run streamlit code for now